Uno.Gallery/Uno.Gallery.Wasm/WasmScripts/UnoGallery.js

```diff
@@ -1,4 +1,49 @@
 var Uno;
+(function (Uno) {
+    var UI;
+    (function (UI) {
+        class FragmentNavigationHandler {
+            static getCurrentFragment() {
+                return window.location.hash;
+            }
+            static setCurrentFragment(fragment) {
+                window.location.hash = fragment;
+                this.currentFragment = window.location.hash;
+                return "ok";
+            }
+            static subscribeToFragmentChanged() {
+                if (this.subscribed) {
+                    return "already subscribed";
+                }
+                this.subscribed = true;
+                this.currentFragment = this.getCurrentFragment();
+                window.addEventListener("hashchange", _ => this.notifyFragmentChanged(), false);
+                return "ok";
+            }
+            static notifyFragmentChanged() {
+                const newFragment = this.getCurrentFragment();
+                if (newFragment === this.currentFragment) {
+                    return; // nothing to do
+                }
+                this.currentFragment = newFragment;
+                this.initializeMethods();
+                const newFragmentStr = MonoRuntime.mono_string(newFragment);
+                MonoRuntime.call_method(this.notifyFragmentChangedMethod, null, [newFragmentStr]);
+            }
+            static initializeMethods() {
+                if (this.notifyFragmentChangedMethod) {
+                    return; // already initialized.
+                }
+                const asm = MonoRuntime.assembly_load("Uno.Gallery.WASM");
+                const handlerClass = MonoRuntime.find_class(asm, "Uno.UI.Wasm", "FragmentHavigationHandler");
+                this.notifyFragmentChangedMethod = MonoRuntime.find_method(handlerClass, "NotifyFragmentChanged", -1);
+            }
+        }
+        FragmentNavigationHandler.subscribed = false;
+        UI.FragmentNavigationHandler = FragmentNavigationHandler;
+    })(UI = Uno.UI || (Uno.UI = {}));
+})(Uno || (Uno = {}));
+var Uno;
```

Comment:
this is a repeated declaration of `Uno`.

Reply:
Fixed. 
https://github.com/unoplatform/Uno.Gallery/commit/26d34686db9cd34a5e7227e8723af669f3c4493e